src/test/java/com/napier/sem/Record_UnitTests.java

```diff
@@ -17,67 +45,43 @@ public void capitalCity_testGettersAndToString()
         Integer population = 6000000;
         String expectedToStringOutput = "Name: Glasgow | Country: Scotland | Population: 6000000\n" +
                 "-------------------------------------------------------";
+        String expectedRecordType = App.CAPITAL_CITY;
 
         // when
-        CapitalCity capitalCity = new CapitalCity(name, country, region, continent, population);
+        Record capitalCity = new Record(name, country, region, continent, population);
 
         // then
         assertEquals(name, capitalCity.getName());
         assertEquals(country, capitalCity.getCountry());
         assertEquals(region, capitalCity.getRegion());
         assertEquals(continent, capitalCity.getContinent());
         assertEquals(population, capitalCity.getPopulation());
+        assertEquals(expectedRecordType, capitalCity.getRecordType());
         assertEquals(expectedToStringOutput, capitalCity.toString());
     }
 
     @Test
-    public void city_testGettersAndToString()
+    public void Record_testGettersAndToStringForCity()
     {
         // given
         String name = "Wishae";
         String country = "Scotland";
         String district = "North Lanarkshire";
         Integer population = 10000;
+        String expectedRecordType = App.CITY;
```

Comment:
Not a massive issue but these are in a different order than in the method above.